src/renderStatsCard.js

```diff
@@ -111,6 +111,9 @@ const renderStatsCard = (stats = {}, options = { hide: [] }) => {
     hide_rank ? 0 : 150
   );
 
+  // Calculate the card width based on whether the rank is displayed or not
+  let width = hide_rank ? name.width : 495;
```

Comment:
Huh? name.width???

Reply:
Yeah not sure what I was thinking, alternatively we could hard code a value in but the name stat changes in width depending on the user input. For some reason I thought you could access the width like that

Reply:
It's a string you can't have a width. 
try `name.length * 5`

Reply:
It seems to be way too small: https://github-readme-stats-git-width-fit-to-content.terror.vercel.app/api?username=terror&hide_rank=true

I am trying to figure out a way for the width of the card to adjust to the length (width) of the longest element on the card if the rank is hidden. 

I just assumed the title element was always the longest however now I realize it can be hidden.

Reply:
We'd have to search through the list of non hidden stats and set the width correspondingly with the longest element